presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -158,23 +158,6 @@ public void testEmptyTableHandleNotCachedWhenCacheMissingIsFalse()
         dropTable(phantomTable);
```

Comment:
Try to place cleanup commits at the beginning of the PR. It is easier to maintain such PRs then. If you get comments to commits in the middle then you might get conflicts. Cleanup commits like conflicts, humans don't.